components/avatar/avatar.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Avatar</title>
+  <link rel="stylesheet" href="avatar.css">
+  <script src="https://kit.fontawesome.com/0b01ac136a.js" crossorigin="anonymous">
+  </script>
+</head>
+
+<body>
+
+  <!-- nav bar -->
+  <nav class="navigation container">
+    <div class="nav-brand">
+      <a href="../../index.html" class="icon"><i class="fas fa-atom"></i></a>
+      Atom UI
+    </div>
+
+    <ul class="list-non-bullet nav-list">
+      <li class="list-item-inline">
+        <a class="link" href="../../index.html">Home</a>
+      </li>
+      <li class="list-item-inline">
+        <a class="link link-active" href="avatar.html">Documentation</a>
+      </li>
+      <li class="list-item-inline">
+        <a class="link" href="https://github.com/sharath-io/Atom-UI">Github</a>
+      </li>
+    </ul>
+  </nav>
+
+  <!-- sidebar -->
+  <aside class="sidebar">
+    <ul>
+      <li><a href="avatar.html">Avatar</a></li>
+      <li><a href="../badge/badge.html">Badge</a></li>
+      <li><a href="./alert.html">Alert</a></li>
+      <li><a href="./button.html">Button</a></li>
+      <li><a href="./card.html">Card</a></li>
+      <li><a href="./image.html">Image</a></li>
+      <li><a href="./input.html">Input</a></li>
+    </ul>
+  </aside>
+
+  <div class="body-text">
+    <h1>Avatars</h1>
+    <p class="description">The Avatar component is used to represent a user, and displays the profile picture, initials
+      or fallback icon.</p>
+    <h2>Square Avatars</h2>
+    <div class="avatar-container flex-center">
+      <img class="avatar size-s" src="https://i.pravatar.cc/400?img=65">
+      <img class="avatar size-m" src="https://i.pravatar.cc/400?img=13">
+      <img class="avatar size-l" src="https://i.pravatar.cc/400?img=2">
+    </div>
+    <p class="description">Add the avatar class to img element.Use the class avatar for avatar component and classes
+      size-s size-m size-l for avatars of different sizes</p>
+
+    <iframe
+      src="https://carbon.now.sh/embed?bg=rgba%2815%2C118%2C110%2C1%29&t=seti&wt=none&l=auto&width=500&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=38px&ph=59px&ln=false&fl=1&fm=Hack&fs=13.5px&lh=162%25&si=false&es=2x&wm=false&code=%2520%253Cimg%2520class%253D%2522avatar%2520size-s%2522%2520src%253D%2522%2520%2522%253E%250A%2520%253Cimg%2520class%253D%2522avatar%2520size-m%2522%2520src%253D%2522%2520%2522%253E%250A%2520%253Cimg%2520class%253D%2522avatar%2520size-l%2522%2520src%253D%2522%2520%2522%253E"
+      style="width: 460px; height: 213px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
You can add a class to iframes so that they can become responsive. This can be done later if you need

Reply:
Will update this later for responsiveness. Thank you for suggestion